CONTRIBUTING.md

```diff
@@ -0,0 +1,194 @@
+# Contributing to Prenda Spark
+
+Thank you for helping us make this project great and being a part of the Prenda Spark community. Here are a few guidelines that will help you along the way.
+
+## Code of Conduct
+
+Prenda Spark has adopted the [Contributor Covenant](https://www.contributor-covenant.org/) as its Code of Conduct, and we expect project participants to adhere to it.
+Please read [the full text](/CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
+
+## Design-to-Development Pipeline
+
+Prenda Spark represents a collaborative effort between the Design and Engineering teams at Prenda. Every component starts out in a Figma Design page within the Web file. Once a component is initially completed, it can be implemented in the Spark repository. Any further changes should be tracked in this repository as well. This requires a significant amount of coordination, but we know it is possible with everyone involved.
+
+### New
+
+1. A designer completes an initial component design.
+2. The designer files a "New Component" Issue in this repository. The description should:
+  - Provide a link to the corresponding Figma page.
+  - Describe the motivation and use-case of component, if necessary.
+  - Enumerate descriptors like variants, sizes, etc.
+  - Describe component makeup, if the component has, or is, a composition pattern.
+3. An engineer assesses the description, communicates with the designer as necessary, and produces an initial written assessment:
+  - Formalize enumerated descriptors to props
+  - Suggest which components from [Material-UI](https://material-ui.com) can be custom-styled or modified to produce the new component.
+4. Label the issue with "ready" (to be worked on).
+5. A developer "claims" the issue (assigns themselves), create a new branch, begins work on implementation.
```

Comment:
**An engineer** and **A Developer** are both the same right? Would it be better to keep as one?

Reply:
Probably.